tests/unittest/auto_deploy/singlegpu/models/test_decilm_modeling.py

```
@@ -0,0 +1,553 @@
+def _create_small_config():
+    """Create a small DeciLM config for testing.
+
+    Uses AutoConfig with trust_remote_code=True to load the HF DeciLMConfig class,
+    then creates a small instance for testing.
+    """
+    try:
+        hf_config = AutoConfig.from_pretrained(
+            "nvidia/Llama-3_3-Nemotron-Super-49B-v1", trust_remote_code=True
+        )
+        ConfigCls = type(hf_config)
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)